SpatialGDK/Documentation/content/toolbar.md

```diff
@@ -1,72 +1,90 @@
-# SpatialOS GDK for Unreal toolbar
+# Toolbars in the Unreal Editor
 
-The GDK toolbar provides several functions required for building and launching your client and server-workers from inside the Unreal Editor.
+There are two toolbars you can use in the Unreal Editor: the main Unreal toolbar, and the SpatialOS for Unreal GDK toolbar. Once enabled, the GDK toolbar sits alongside the main Unreal toolbar:
```

Comment:
"SpatialOS GDK for Unreal", or simply "the GDK"

Reply:
I'd say "SpatialOS GDK toolbar" ? (From memory, we use this in the docs and "GDK toolbar" once we have mentioned SpaitalOS once. :-) )